inc/Engine/Cache/PurgeActionsSubscriber.php

```diff
@@ -54,11 +55,44 @@ public function purge_user_cache( $user_id ) {
 		rocket_clean_user( $user_id );
 	}
 
+	/**
+	 * Purges the cache when a public term is created|updated|deleted
+	 *
+	 * @since 3.5.5
+	 *
+	 * @param int    $term_id  Term ID.
+	 * @param int    $tt_id    Term taxonomy ID.
+	 * @param string $taxonomy Taxonomy slug.
+	 * @return void
+	 */
+	public function maybe_purge_cache_on_term_change( $term_id, $tt_id, $taxonomy ) {
+		if ( ! $this->is_taxonomy_public( $taxonomy ) ) {
+			return;
+		}
+
+		rocket_clean_domain();
+	}
+
+	/**
+	 * Checks if the given taxonomy is public
+	 *
+	 * @param string $name Taxonomy name.
+	 * @return bool
+	 */
+	private function is_taxonomy_public( $name ) {
+		$taxonomy = get_taxonomy( $name );
+
+		if ( ! $taxonomy ) {
```

Comment:
`get_taxonomy()` returns `WP_Taxonomy` or `false`. The guard clause here should check if `false` is returned. Right now, it's a boolean check that evaluates whatever data type is returned from `get_taxonomy()`. If an object of `WP_Taxonomy` is returned, PHP has to type juggle to do the boolean check. Instead, we can be a bit more clear and performant by checking:

```php
if ( false === $taxonomy ) {
```

When doing this, it avoids the type juggle and is a little faster.